src/calculator.js

```diff
@@ -1,36 +1,36 @@
 "use strict";
 
-var Precedence = require('./precedence');
-var Screen = require('./screen');
-var Memory = require('./memory');
+const Precedence = require('./precedence');
```

Comment:
ES6 offers `import` directive, which is a new way do organise imports. Please investigate and change in all modules.